coral/src/app/features/approvals/acls/AclApprovals.tsx

```diff
@@ -263,24 +327,61 @@ function AclApprovals() {
   ];
 
   return (
-    <ApprovalsLayout
-      filters={filters}
-      table={
-        <DataTable
-          ariaLabel={"Acl requests"}
-          columns={columns}
-          rows={tableData}
-          noWrap={false}
-        />
-      }
-      pagination={
-        <Pagination
-          activePage={data.currentPage}
-          totalPages={data.totalPages}
-          setActivePage={handleChangePage}
+    <>
+      {detailsModal.isOpen && (
+        <RequestDetailsModal
+          onClose={() => setDetailsModal({ isOpen: false, reqNo: "" })}
+          onApprove={() => {
+            approveRequest({ req_no: detailsModal.reqNo });
+          }}
+          onReject={() => {
+            setDetailsModal({ isOpen: false, reqNo: "" });
+            setRejectModal({ isOpen: true, reqNo: detailsModal.reqNo });
+          }}
+          isLoading={approveIsLoading}
+        >
+          {
+            <DetailsModalContent
+              aclRequest={data.entries.find(
+                (request) => request.req_no === Number(detailsModal.reqNo)
+              )}
+            />
+          }
+        </RequestDetailsModal>
+      )}
+      {rejectModal.isOpen && (
+        <RequestRejectModal
+          onClose={() => setRejectModal({ isOpen: false, reqNo: "" })}
+          onCancel={() => setRejectModal({ isOpen: false, reqNo: "" })}
+          onSubmit={(message: string) => {
+            rejectRequest({
+              req_no: rejectModal.reqNo,
+              reasonForDecline: message,
+            });
+          }}
+          isLoading={rejectIsLoading}
         />
-      }
-    />
+      )}
+      {errorMessage !== "" && <Alert type="warning">{errorMessage}</Alert>}
```

Comment:
Nothing to change her, just remark: in this cases I like `Boolean(errorMessage)` because I feel it's a bit more explicit.

Reply:
Oh, but note about `<Alert />` - it's not accessible. We need to wrap it in e.g. here

```
<div role="alert">
```

Reply:
> Nothing to change her, just remark: in this cases I like Boolean(errorMessage) because I feel it's a bit more explicit.

Ah, that's interesting, because I would think that your way would make it less legible ^^' I always prefer to check on the original type of a value instead of doing coercions, either explicit or implict 🤔

Reply:
>Ah, that's interesting, because I would think that your way would make it less legible ^^ 
😬  

Yes, checking the original type is more explicit in a way, that's true. 

Thanks for wrapping! 👯  In that case the fix to make it (more) accessible is very easy. That's the kind of fix I like 😅

Reply:
[572bb31](https://github.com/aiven/klaw/pull/629/commits/572bb31269cb01222e7b3db58f63665cbf31850c)